command/src/config.rs

```diff
@@ -214,13 +215,66 @@ pub struct FileAppConfig {
   pub certificate:       Option<String>,
   pub key:               Option<String>,
   pub certificate_chain: Option<String>,
-  pub backends:          Vec<String>,
+  pub backends:          Vec<BackendConfig>,
   pub sticky_session:    Option<bool>,
   pub https_redirect:    Option<bool>,
   #[serde(default)]
   pub send_proxy:        Option<bool>,
   #[serde(default)]
   pub expect_proxy:      Option<bool>,
+  #[serde(default)]
+  pub lb_policy: LoadBalancingAlgorithms,
```

Comment:
why not an `Option<LoadBalancingAlgorithms>` here? will it make a `LoadBalancingAlgorithms::RoundRobin` when the field is not present?